protostar/testing/cheatcodes/send_message_to_l2.py

```diff
@@ -0,0 +1,68 @@
+from typing import Callable, Any, Mapping, Optional
+
+from starkware.starknet.business_logic.execution.execute_entry_point import (
+    ExecuteEntryPoint,
+)
+from starkware.starknet.business_logic.execution.objects import CallType
+from starkware.starknet.public.abi import get_selector_from_name
+from starkware.starknet.services.api.contract_class import EntryPointType
+
+from protostar.starknet import Cheatcode, CheatcodeException
+from protostar.utils.data_transformer import CairoOrPythonData, from_python_transformer
+
+
+class SendMessageToL2Cheatcode(Cheatcode):
+    @property
+    def name(self) -> str:
+        return "send_message_to_l2"
+
+    def build(self) -> Callable[..., Any]:
+        return self.send_message_to_l2
+
+    def send_message_to_l2(
+        self,
+        fn_name: str,
+        l1_sender_address: int = 0,
+        contract_address: Optional[int] = None,
+        calldata: Optional[CairoOrPythonData] = None,
```

Comment:
[Functionality, Consistency] I think the signature should be similar to `starkware.starknet.testing.starknet.Starknet::send_message_to_l2`